File: scripts/py-compile-suite/inputs/28_continuation_comments.m
%% Section A: Setup
% Initialize parameters
N = 1000;       % number of samples
fs = 44100;     % sample rate in Hz
f = 440;        % frequency in Hz

%% Section B: Generate signal
t = (0:N-1) / fs;
signal = sin(2*pi*f*t) + ...
         0.5*sin(2*pi*2*f*t) + ...
         0.25*sin(2*pi*3*f*t);

%% Section C: Compute RMS
rms_val = sqrt(mean(signal.^2));

File: scripts/py-compile-suite/outputs/28_continuation_comments.py
import numpy as np

#% Section A: Setup
# Initialize parameters
N = 1000
# number of samples
fs = 44100
# sample rate in Hz
f = 440
# frequency in Hz

#% Section B: Generate signal
t = np.arange(0, N-1 + 1) / fs
signal = np.sin(2*np.pi*f*t) + 0.5*np.sin(2*np.pi*2*f*t) + 0.25*np.sin(2*np.pi*3*f*t)

#% Section C: Compute RMS
rms_val = np.sqrt(np.mean(signal**2))
